matter/wifi/wf200/sl_wfx_board.h

```diff
@@ -8,8 +8,10 @@
 #elif defined(EFR32MG12_BRD4161A) || defined(BRD4161A) || defined(EFR32MG12_BRD4163A) || defined(BRD4163A) \
   || defined(EFR32MG12_BRD4164A) || defined(BRD4164A)
 #include "brd4161a.h"
-#elif defined(EFR32MG24_BRD4187A) || defined(BRD4187A)
+#elif defined(EFR32MG24_BRD4187C) || defined(BRD4187C)
```

Comment:
REV A should still be supported. this will cause some backward compatibility issue. @jmartinez-silabs what do you think ?

Reply:
I'll have to look at what is inside their brd4186C.h maybe it is usable for both. If it is the case then I would add the REV A in those #ifdef as there are slight changes in the bring-up configuration for both revs.

It is preferable to build a revA and revC distinctly

Reply:
We are not adding support for revA boards , please add support.

Reply:
yes, we don't have revA boards and we don't have pin changes also. , please add defines/header for revA boards when required.